src/main/java/teammates/common/datatransfer/questions/FeedbackResponseDetails.java

```diff
@@ -31,6 +31,10 @@ public abstract void extractResponseDetails(
 
     public abstract String getAnswerHtml(FeedbackQuestionDetails questionDetails);
 
+    public String getDetailedAnswerHtml(FeedbackQuestionDetails questionDetails) {
+        return getAnswerHtml(questionDetails);
+    }
```

Comment:
This design looks more understandable. `getAnswerHtml` should be changed in several other places. But I think its okay and it can be done pretty easily.